transform.go

```diff
@@ -63,18 +63,27 @@ type OpType int
 
 const (
 	OpTypeEquals OpType = iota
+	OpTypeNotEquals
 	OpTypeLessThan
-	OpTypeGreaterEquals
+	OpTypeLessEqual
+	OpTypeGreaterThan
+	OpTypeGreaterEqual
 	OpTypeIn
 )
 
 func opTypeToString(value OpType) string {
 	switch value {
 	case OpTypeEquals:
 		return "eq"
+	case OpTypeNotEquals:
+		return "neq"
 	case OpTypeLessThan:
 		return "lt"
-	case OpTypeGreaterEquals:
+	case OpTypeLessEqual:
```

Comment:
nit-pick: Did you want to make it consistently "*Equals"?
i.e. LessEquals, NotEquals, GreaterEquals... whereas right now it's:
NotEqual**s**, LessEqual, GreaterEqual
It's fine as is, though.

Reply:
I actually struggled with what to do here.  This ticket however targets the specific task of adding additional expressions, not renaming old expressions.  I was trying to keep the commits atomic and independent.  I'll change the name of the existing ones in a commit before this one and push it up though.

Reply:
Done.